src/pages/get-involved/join-our-team.tsx

```diff
@@ -25,16 +48,124 @@ export default function JoinOurTeamPage({
 
   return (
     <>
+      {/* Header */}
+      <Section backgroundColor="tertiary.light">
+        <Grid container justifyContent="space-between">
+          {/* Header text + button */}
+          <Grid xs={12} md={6}>
+            <Stack spacing={6} alignItems={{ xs: "center", md: "self-start" }} padding={{ xs: 2, lg: 12 }}>
+              <Typography variant="h3" color="black" textAlign={{ xs: "center", md: "left" }}>
+                {sanityJoinOurTeamPage?.mainHeader}
+              </Typography>
+              <Typography variant="h6" color="black" textAlign={{ xs: "center", md: "left" }}>
+                {sanityJoinOurTeamPage?.subHeader}
+              </Typography>
+              <SanityButton
+              isAnimated
+              boopProps={{ scale: 1.1 }}
+              content={sanityJoinOurTeamPage.headerButton}>
+              </SanityButton>
+            </Stack>
+          </Grid>
+          <Grid md={6} sx={{ display:{ xs: "none", md: "flex" } }} justifyContent="center" alignItems="center">
+            <SanityImage imageAsset={sanityJoinOurTeamPage.headerImage}>
+            </SanityImage>
+          </Grid>
+        </Grid>
+      </Section>
+      {/* Why we volunteer section */}
+      <Section backgroundColor="secondary.light">
+        <Grid container alignItems="stretch" spacing={4}>
+          <Grid xs={12}>
+            <Stack spacing={6} alignItems="center">
+              <Typography variant="h3" textAlign="center">
+                Why do we volunteer?
```

Comment:
This is also something that can be pulled from Sanity, but agreed this is fine for now!